docs/atomic-diff.1.md

```diff
@@ -63,6 +69,10 @@ Compare the files and RPMs (without versions) in images 'foo1' and 'foo2' and ou
 Compare only the metadata between images 'foo1' and 'foo2'
 
     atomic diff -m foo1 foo2
+    
+Compare files by 'sha256digests' and 'time' between images 'foo1' and 'foo2'
+
+    atomic diff foo1 foo2 --keywords sha256digest time
```

Comment:
@rhatdan 

It only works if if the keywords are the last arguments.  Verified this in argparse as well.  Example:

```
[bbaude@bbaude atomic (diff_mtree)]$ sudo ./atomic diff -k sha256digest centos:latest 19f8d8b7147b
atomic diff: argument -k/--keywords: invalid choice: 'centos:latest' (choose from 'all', 'link', 'nlink', 'mode', 'type', 'time', 'uid', 'gid', 'size', 'sha256digest')
Try 'atomic diff --help' for more information.
```
^^ Fails.  Keywords have to be last.
```
[bbaude@bbaude atomic (diff_mtree)]$ sudo ./atomic diff centos:latest 19f8d8b7147b -k sha256digest
```
Ideas and/or suggestions welcome.